security/audit-log-policy-config.adoc

```diff
@@ -1,14 +1,14 @@
-[id="nodes-nodes-audit-config"]
-= Configuring the node audit log policy
+[id="audit-log-policy-config"]
+= Configuring the audit log policy
 include::modules/common-attributes.adoc[]
-:context: nodes-nodes-audit-config
+:context: audit-log-policy-config
 
 toc::[]
 
-You can control the amount of information that is logged to the node audit logs by choosing the audit log policy profile to use.
+You can control the amount of information that is logged to the audit logs by choosing the audit log policy profile to use.
```

Comment:
would be more specific and call it "API server audit logs". It's not system audit.